build_matrix_A.py

```diff
@@ -0,0 +1,86 @@
+import numpy as np
+
+
+def build_matrix_A(material_property_library, mesh):
+    """ This function will take a previously generated mesh and create the 'A'
+    matrix used in solving this problem.
+
+    inputs:
+    --------
+    - material_property_library: this will be a dictionary or nested dictionary
+    containing all of the important data for each material (fuel,
+    various layers of graphite)
+    - mesh: the physical location of each mesh point
+    - materials: the materials in each region and how many nodes are assigned to that material
+    outputs. List of tuples of format (material, number of nodes)
+    --------
+    - An array that can be plugged into a subsequent function to solve the
+    A.x=b formula
+    """
+    k = np.zeros(len(mesh))
+    rho = np.zeros(len(mesh))
+    c = np.zeros(len(mesh))
+    DR = np.zeros(len(mesh))
+    for m in range(len(mesh)):
+        k[m] += material_property_library[mesh[m][0]]['k']
```

Comment:
Is the `+=` operator necessary here? It seems like `=` would be fine since it appears that you are only accessing each `m` once.